Sources/Solita/RenderInstruction.swift

```diff
@@ -47,7 +47,7 @@ public class InstructionRenderer {
             fieldName: "instructionDiscriminator",
             typeMapper: typeMapper
         )
-        self.programIdPubkey = "PublicKey(string: \"\(self.programId)\")!"
+        self.programIdPubkey = "PROGRAM_ID!"
```

Comment:
This is generated an incorrect code in generation
![Screenshot 2022-11-14 at 12 23 49 PM](https://user-images.githubusercontent.com/316694/201737070-2b74d285-571e-4423-b9ed-49628d456121.jpg)

Reply:
Fixed in latest push.